sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedValuesReader.java

```diff
@@ -86,4 +87,26 @@ void readLongsWithRebase(
     void write(WritableColumnVector outputColumnVector, int rowId, long val);
   }
 
+  @FunctionalInterface
+  interface ByteBufferOutputWriter {
+    void write(WritableColumnVector c, int rowId, ByteBuffer val, int length);
+
+    static void writeArrayByteBuffer(WritableColumnVector c, int rowId, ByteBuffer val,
```

Comment:
Is it a good practice to add `static` methods to interface?

Reply:
I don't know if it is frowned upon. In this case, not including in the interface only leads to some code bloat.